extensions/wikia/Insights/models/InsightsQuerypageModel.php

```diff
@@ -528,6 +532,14 @@ public function getNextItem( $type, $articleName ) {
 		return $next;
 	}
 
+	public function purgeInsightCache() {
```

Comment:
Minor thing, but I think that we should call it `purgeInsightsCache` (plural form) since the update method also uses it (`updateInsightsCache`). :)